jwst/guider_cds/tests/test_guider_cds.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+import pytest
+
+from jwst import datamodels
+from jwst.guider_cds.guider_cds import get_dataset_info, guider_cds
+
+
+@pytest.fixture(scope='module')
+def make_guider_image():
+    """Generate science image"""
+
+    image = datamodels.GuiderRawModel()
+
+    image.meta.instrument.name = 'FGS'
+    image.meta.exposure.frame_time = 234.3423235
+    image.meta.exposure.ngroups = 4
```

Comment:
It looks as if ngroups is actually 10, based on the image shape (it's the number of integrations that's 4).  It isn't used except for a log message, so it probably doesn't matter.